site/profile/manifests/core/icinga_master.pp

```diff
@@ -180,6 +183,8 @@
       },
     },
   }
+  include ::icinga2::pki::ca
```

Comment:
I presume this results in a new CA being dynamically generated?  What happens if the master is reinstalled?  Will all the agents break?

Reply:
I think you are right, they should break, but as Heinrich said they should renogotiate

Reply:
if the master is reinstall, all agents renegociate